test/integ/quote-classic.test.ts

```diff
@@ -2218,6 +2218,8 @@ describe('quote', function () {
     [ChainId.MOONBEAM]: null,
     [ChainId.GNOSIS]: null,
     [ChainId.ARBITRUM_GOERLI]: null,
+    [ChainId.BASE_GOERLI]: USDC_ON(ChainId.BASE_GOERLI),
+    [ChainId.BASE]: USDC_ON(ChainId.BASE),
```

Comment:
I thought USDC doesn't exist on Base?

Reply:
I'm not sure lol, this is copied over from routing-api

Reply:
it exists now